addons/website/static/src/scss/website.scss

```diff
@@ -1464,6 +1464,13 @@ span.list-inline-item.o_add_language:last-child {
         position: relative;
         z-index: 1;
     }
+    footer#bottom:not(:has(.o_footer_copyright)) {
+        #o_footer_scrolltop_wrapper {
+            height: 3rem !important;
```

Comment:
Instead of relying on `:has(.o_footer_copyright)`, I think a more general selector would be `#o_footer_scrolltop_wrapper:last-child`, considering this is the real reason for the issue.
And then you can just nest it one line above. :)

Reply:
you meant `o_footer:last-child` right ?